sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_detect_change_point.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+FILE: sample_detect_change_point.py
+
+DESCRIPTION:
+    This sample demonstrates how to detect entire series change points.
+
+Prerequisites:
+     * The Anomaly Detector client library for Python
+     * A .csv file containing a time-series data set with
+        UTC-timestamp and numerical values pairings.
+        Example data is included in this repo.
+
+USAGE:
+    python sample_detect_change_point.py
+
+    Set the environment variables with your own values before running the sample:
+    1) ANOMALY_DETECTOR_KEY - your source Form Anomaly Detector API key.
+    2) ANOMALY_DETECTOR_ENDPOINT - the endpoint to your source Anomaly Detector resource.
+"""
+
+import os
+
+
+class DetectChangePointsSample(object):
+
+    def detect_change_point(self):
+        from azure.ai.anomalydetector import AnomalyDetectorClient
```

Comment:
This is not recommended to do import inside a function, and this would not look Pythonic to our readers

Reply:
done